spa_ui/self_service/client/assets/sass/_details-view.sass

```diff
@@ -0,0 +1,53 @@
++block(ss-details-header)
+  border-bottom: 1px solid #f2f2f2
+  margin-bottom: 20px
+
+  +element(title-img)
+    background: $app-color-white
+    border: 1px solid $app-color-light-gray-4
+    box-shadow: 0 1px 1px rgba($app-color-black, 0.175)
```

Comment:
without nitpicking, I would encourage the use the `lighten()` or `darken()` functions of sass instead of rbga, (far more fantastically readable!)